packages/cloud-cognitive/src/components/ComboButton/ComboButton.stories.js

```diff
@@ -17,19 +16,112 @@ import { ComboButton, ComboButtonItem } from '..';
 
 import styles from './_combo-button.scss';
 
+const modifiers = {
+  'Is Delete': { isDelete: true },
+  'No Delete': { isDelete: false },
+  'Has Divider': { hasDivider: true },
+  'No Divider': { hasDivider: false },
+  'Item Disabled': { itemDisabled: true },
+  'Item Active': { itemDisabled: false },
+};
 export default {
   title: getStoryTitle(ComboButton.displayName),
   component: ComboButton,
+  argTypes: {
+    size: {
+      control: {
+        type: 'radio',
+      },
+      options: ['sm', 'md', 'lg'],
+    },
+    // }
+    modifiers: {
+      control: {
+        type: 'select',
+        labels: Object.keys(modifiers),
+      },
+      options: Object.values(modifiers).map((_k, i) => i),
+      mapping: Object.values(modifiers),
+    },
+  },
   subcomponents: {
     ComboButtonItem,
   },
   parameters: { styles },
 };
 
-export const Default = prepareStory(() => (
-  <ComboButton>
-    <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
-    <ComboButtonItem renderIcon={CloudApp16}>ComboButtonItem 2</ComboButtonItem>
-    <ComboButtonItem>ComboButtonItem 3</ComboButtonItem>
-  </ComboButton>
-));
+const Template = (args) => {
+  console.log(args);
+  return (
+    <ComboButton disabled={args.buttonDisabled} size={args.size}>
+      <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
```

Comment:
This one’s already in the Storybook, but I actually recommend we rename this to “Primary button” for more consistency with the way we usually refer to actions.